### PR TITLE
WIP: golint-fixes: pkg/kubectl/cmd

### PR DESCRIPTION
**Pull request by sayanarijit**
**What this PR does / why we need it**:
To fix/minimize golint failures. Issue: #68026

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:

**Special notes for your reviewer**:
I am working on `pkg/kubectl/cmd` package. Please review changes and let me know if I am doing it right.

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by nikhita** [comment]
@sayanarijit Thanks for your first contribution to Kubernetes! :tada: 

> I am working on pkg/kubectl/cmd package. 

This PR touches other packages as well. Can you split other packages into separate commits or PRs so that it's easier to review? 222 files changed is a lot of changes. :)

**Comment by sayanarijit** [comment]
@nikhita Thanks for your feedback. I touched only `kubelet/cmd` and `cluster/images`. Others are changed automatically (by `go-fmt`).
It's my mistake to commit all changes together.
I will remove other changes from this PR and let you know.

**Comment by sayanarijit** [comment]
@nikhita I have reduces to to 91 now. Please let me know if it is okay. Not sure why the bot marked me as non cncf member.
I can also create a fresh PR as it got a lot of unnecessary badges. I have my changes already backed up.

File: pkg/kubectl/cmd/annotate.go [hunk 1]
```diff
@@ -41,39 +41,6 @@ import (
 	"k8s.io/kubernetes/pkg/kubectl/util/i18n"
 )
 
-// AnnotateOptions have the data required to perform the annotate operation
-type AnnotateOptions struct {
```
Comment (yue9944882) [inline pkg/kubectl/cmd/annotate.go]: i see several definition moved like this. is it necessary?

Comment (sayanarijit) [reply pkg/kubectl/cmd/annotate.go]: Hi Kim
It was not necessary but while editing, I observed the pattern of defining vars first, then structs. So I moved them to follow the pattern. Shall I revert them?

Comment (yue9944882) [reply pkg/kubectl/cmd/annotate.go]: > Shall I revert them?

yeah. especially for such "wide" pulls. not only easier for review, but also helpful for sorting blame msg. plz keep a minimal change.

Comment (sayanarijit) [reply pkg/kubectl/cmd/annotate.go]: Done.
Please review and let me know if there's something else I should keep in mind.

**Comment by sayanarijit** [comment]
@nikhita @yue9944882 
Last 2 packages remaining in `pkg/kubectl/cmd`. However having trouble with `	pkg/kubectl/cmd/util/openapi`.

In this file there is an import `"k8s.io/kubernetes/pkg/kubectl/cmd/util/openapi"` which is basically itself.
It also has few interfaces missing and there are type mismatch errors everywhere.

Can anyone suggest what to do with it?

**Comment by yue9944882** [comment]
hi @sayanarijit thanks for contributing. but this pull is getting far bigger than i thought. could u plz divide this pull into smaller pulls so that our reviewers can review this better? for one pull, hundred-ish lines of changes might be more reasonable.

> It also has few interfaces missing and there are type mismatch errors everywhere.

seems u missed some generated code. did u run `make`?

**Pull request closed without merging**